test/new_relic/agent/log_event_aggregator_test.rb

```diff
@@ -324,6 +328,35 @@ def test_records_metrics_on_harvest
       end
     end
 
+    def test_seen_and_sent_zeroed_out_on_reset!
+      with_config(CAPACITY_KEY => 3) do
+        3.times { @aggregator.record('Are you counting this?', 'DEBUG') }
+        @aggregator.harvest! # seen/sent are counted on harvest, this zeroes things out
+
+        assert_metrics_recorded_exclusive({
+          'Logging/lines' => {:call_count => 3},
+          'Logging/lines/DEBUG' => {:call_count => 3},
+          'Logging/Forwarding/Dropped' => {:call_count => 0},
+          'Supportability/Logging/Forwarding/Seen' => {:call_count => 3},
+          'Supportability/Logging/Forwarding/Sent' => {:call_count => 3}
+        },
+          :ignore_filter => %r{^Supportability/API/})
+
+        @aggregator.record('Are you counting this?', 'DEBUG')
+        @aggregator.reset! # set seen/sent to zero, throwing out the latest #record
+        @aggregator.harvest! # nothing new should be available to count, so use same numbers as before
+
+        assert_metrics_recorded_exclusive({
+          'Logging/lines' => {:call_count => 3},
+          'Logging/lines/DEBUG' => {:call_count => 3}, # ?
```

Comment:
why the ? comment?

Reply:
ooh, forgot about this one, good catch. I felt skeptical that those were the correct numbers, but since we're resetting before harvesting, they are the correct numbers.